examples/images/README.md

```diff
@@ -0,0 +1,12 @@
+Example showing how to show images with eframe/egui.
+
+```sh
+cargo run -p images
+```
+
+You can submit `file://` and `https://` URIs into the text box to load the image in the viewer.
+
+Try loading [ferris.svg](./ferris.svg) using the `file…` dialog!
+
+![](screenshot0.png)
+![](screenshot1.png)
```

Comment:
outdated screenshots

Reply:
also, maybe we can adjust the example to keep the file size of the screenshots down a bit

Reply:
I updated the screenshot and made it much smaller